src/components/ha-tabs.ts

```diff
@@ -36,10 +36,11 @@ class HaTabs extends PaperTabsClass {
 
   _tabChanged(tab, old) {
     super._tabChanged(tab, old);
-    
+
     // Get first and last tab's width for _affectScroll
-    this.firstTabWidth = this.firstElementChild.clientWidth;
-    this.lastTabWidth = this.lastElementChild.clientWidth;
+    const tabs = this.querySelectorAll("paper-tab:not(.hide-tab)");
+    this.firstTabWidth = tabs[1].clientWidth;
```

Comment:
Should be the first right?
```suggestion
    this.firstTabWidth = tabs[0].clientWidth;
```